### PR TITLE
Some increase coverage

### PR DESCRIPTION
**Pull request by joncasdam**
Not that fancy.

### CODE REVIEW

**Comment by JonasSteur** [comment]
@joncasdam can this be merged already or are you going to add more commits ?

**Comment by joncasdam** [comment]
I might add more, yes.

File: wingmonkey/tests/test_members.py [hunk 1]
```diff
@@ -190,3 +190,13 @@ def test_member_serializer_without_session():
...
     assert serializer.session == session
+
+
+def test_generate_member_id_without_email_address():
+    assert not generate_member_id('')
+
+
+def test_generate_member_id():
+    email = 'some@email.com'
+    expected = generate_member_id(email_address=email)
```
Comment (JonasSteur) [inline wingmonkey/tests/test_members.py]: I would put the md5 hash of some@email.com here instead (this is the way mailchimp generates member id's). The way it's set up now it only tests if this function gives the same output on the same input twice in a row but not if it actually correctly creates a member id.

**Comment by joncasdam** [comment]
Now I think I've sent everything I did during the weekend :) I pushed today while current tickets here gave me some time between a building here and there 👍

File: wingmonkey/tests/test_members.py (hunk 1 above)
Comment (joncasdam) [reply wingmonkey/tests/test_members.py]: oh ok, makes sense. I've added a fixup. If it's ok, I'll squash.

Comment (JonasSteur) [reply wingmonkey/tests/test_members.py]: all good now ( besides a small typo I spotted hase instead of hash)
Lets squash it and I'll merge it in 👍

Comment (joncasdam) [reply wingmonkey/tests/test_members.py]: haha true. fixed before squashing

**Pull request merged**